.github/workflows/labeler.yaml

```diff
@@ -0,0 +1,52 @@
+name: Auto Label Issues and PRs
+
+on:
+  issues:
+    types: [opened, edited, reopened, transferred]
+  issue_comment:
+    types: [created, edited]
+  pull_request_target:
+    types: [opened, edited, reopened]
+
+jobs:
+  label:
+    permissions:
+      contents: read
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
+      - name: Get changed files
+        if: ${{ github.event.pull_request }}
+        id: changed-files
+        uses: tj-actions/changed-files@ed68ef82c095e0d48ec87eccea555d944a631a4c
```

Comment:
love that we're pinning to the commit SHA for this... How often do we plan to review this and update it?  Do we have an update policy for pinned dependencies?

Reply:
cool - just want to make sure we have a regular cadence on this so we dont inadvertently get caught up in a vulnerability because we're grabbing latest (even though our risk exposure is probably low)

Reply:
You can use dependabot for this -- it can send PRs for these if you have a comment like `# @v46`, at least.

I can send a PR for dependabot configuration if desired.